Makefile

```diff
@@ -34,7 +34,7 @@ start: ## Start the containers
 stop: ## Stop the containers
 	${DOCKER_COMPOSE} down
 
-wait-healthy: ## Wait for the containers to be healthy
+wait-healthy: gitmodules ## Wait for the containers to be healthy
```

Comment:
Isn't related?

Reply:
Travis CI doesn't require this because it checks out Git submodules on its own, whereas the `actions/checkout` Github Action doesn't so needs this as a dependency

Reply:
Oh hang on, script's in the submodule.

Reply:
Extract or leave?

Reply:
Extract

Reply:
In fact, looks like it can do the same as Travis? https://github.com/actions/checkout#checkout-private-submodules